sim/rogue/envenom.go

```diff
@@ -13,7 +13,7 @@ func (rogue *Rogue) registerEnvenom() {
 	}
 
 	baseAbilityDamage := rogue.baseRuneAbilityDamage()
-
+	consumed := int32(0)
```

Comment:
We should set this at the top of the `ApplyEffects` just to be sure that there's no leakage across iterations